ReadingList/ReadingListBatchRecordResponse.swift

```diff
@@ -11,35 +11,35 @@ class ReadingListBatchRecordResponse: ReadingListResponse {
             for response in responses {
-                if let body = response.valueForKeyPath("body") as? [String:AnyObject] {
-                    if let statusCode = response.valueForKeyPath("status") as? Int {
-                        if let path = response.valueForKeyPath("path") as? String {
-                            if let url = NSURL(string: path, relativeToURL: self.response.URL) {
-                                if let headers = response.valueForKeyPath("headers") as? [String:String] {
-                                    if let r = NSHTTPURLResponse(URL: url, statusCode: statusCode, HTTPVersion: "1.1", headerFields: headers) {
-                                        if let recordResponse = ReadingListRecordResponse(response: r, json: body) {
-                                            self.responses.append(recordResponse)
-                                        } else {
-                                            return nil // This will
-                                        }
-                                    } else {
-                                        return nil // hopefully
-                                    }
-                                } else {
-                                    return nil // become
-                                }
-                            } else {
-                                return nil // much
-                            }
-                        } else {
-                            return nil // nicer code
-                        }
-                    } else {
-                        return nil // when we
-                    }
-                } else {
-                    return nil // use Swift 1.2
+                guard let body = response.valueForKeyPath("body") as? [String:AnyObject] else {
+                    return nil
                 }
+                
+                guard let statusCode = response.valueForKeyPath("status") as? Int else {
+                    return nil
+                }
+                
+                guard let path = response.valueForKeyPath("path") as? String else {
+                    return nil
+                }
+                
+                guard let url = NSURL(string: path, relativeToURL: self.response.URL) else {
+                    return nil
+                }
+                
+                guard let headers = response.valueForKeyPath("headers") as? [String:String] else {
+                    return nil
+                }
+                
+                guard let r = NSHTTPURLResponse(URL: url, statusCode: statusCode, HTTPVersion: "1.1", headerFields: headers) else {
+                    return nil
+                }
+                
+                guard let recordResponse = ReadingListRecordResponse(response: r, json: body) else {
+                    return nil
+                }
+                
+                self.responses.append(recordResponse)
```

Comment:
You can collapse all of these guard statements into a single statement since they all return the same thing:

```swift
guard let body = response.valueForKeyPath("body") as? [String:AnyObject],
      let statusCode = response.valueForKeyPath("status") as? Int,
      let path = response.valueForKeyPath("path") as? String,
      let url = NSURL(string: path, relativeToURL: self.response.URL),
      let headers = response.valueForKeyPath("headers") as? [String:String], 
      let r = NSHTTPURLResponse(URL: url, statusCode: statusCode, HTTPVersion: "1.1", headerFields: headers),
      let recordResponse = ReadingListRecordResponse(response: r, json: body) else {
    return nil
}
```